impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -193,30 +193,16 @@ static constexpr vk::ImageUsageFlags ToVKImageUsageFlags(
       vk_usage |= vk::ImageUsageFlagBits::eDepthStencilAttachment;
     } else {
       vk_usage |= vk::ImageUsageFlagBits::eColorAttachment;
+      vk_usage |= vk::ImageUsageFlagBits::eInputAttachment;
     }
-    vk_usage |= vk::ImageUsageFlagBits::eInputAttachment;
```

Comment:
We don't attach depth/stencil, only the color textures (actually, only the MSAA one, maybe I can fix that too).